source/_posts/binarized-neural-network.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Binary Neural Network “A New Revolution in AI”?
```

Comment:
use 3-4 some keyword from this list 

NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION, 
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING

Reply:
This is the research blog... I have added Nayan and AI keyword in it